### PR TITLE
Basic conversion to setup.cfg

### PR DESCRIPTION
**Pull request by jeromekelleher**

### CODE REVIEW

**Review by benjeffery — APPROVED** [review]
LGTM!

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel",
+    "numpy",
```
[the remark this answers is not in the document]
Comment (jeromekelleher) [reply pyproject.toml]: Aha, thanks @grahamgower! I've used [oldest_supported_numpy](https://pypi.org/project/oldest-supported-numpy/) instead, which does the same thing. Look OK to you?

[the remark this answers is not in the document]
Comment (grahamgower) [reply pyproject.toml]: What an excellent package! I can't believe I didn't find this during my adventures trying to deal with this numpy madness last month.

[the remark this answers is not in the document]
Comment (grahamgower) [reply pyproject.toml]: Pinging @apragsdale, in case he wants to do this in `moments` too.

**Pull request merged**